src/HostManager.cpp

```diff
@@ -38,39 +38,34 @@
 #include "post_guard.h"
 
 
-bool HostManager::deleteHost( QString hostname )
+bool HostManager::deleteHost(QString hostname)
 {
     qDebug() << "HostManager::deleteHost(" << hostname.toUtf8().constData() << ") INFO: trying to delete host from host pool, getting write lock...";
```

Comment:
As of Qt 5.6 we have `void QDebug::setVerbosity(int verbosityLevel)` but I'm not sure how that works for the _built-in_ non-member of the `QDebug` class (i.e. uses where we send standard Qt objects to that class) - note this is not the same as user provided ones, like my `TEvent` one as that could be revised to make use of `QDebug::verbosity` or rather the public getter of that presumably private member.